setup-script.py

```diff
@@ -0,0 +1,17 @@
+from pybars import Compiler
+import os
+
+DB_MANIFEST_TEMPLATE = 'k8s/running/templates/mariadb.tmpl'
+DB_MANIFEST = 'k8s/running/mariadb.yaml'
```

Comment:
I think we probably want to just have a directory of files. If any end with `.tmpl` they're templates and we should run handlebars on them. All should be `kubectl apply`'d. That makes it easier to add stuff without needing to also update the script.

Reply:
I like that idea. I've removed the templates directory and put the `.tmpl` files in their respective directories.